README.md

```diff
@@ -18,64 +17,75 @@ Then, add one or more of the modules below:
 
 The project consists of three modules that are independent of each other, comprising together a complete solution.
 
-### 1. Serializer
+### 1. Checker Plugin
 
-Simple Borer based Akka serializer. It uses codecs, provided by Borer, that are generated during compile time (so
-serializer won't crash during runtime like reflection-based serializers may do).
+A Scala compiler plugin that detects messages, events etc. and checks, whether they extend the base trait and reports
```

Comment:
```suggestion
A Scala compiler plugin that detects messages, events and persistent state, and checks whether they extend the base trait and reports
```

plus a link to each of 3 categories (esp. given that it's the first they're mentioned)

Reply:
Not sure I there is a link to add. These terms are defined in documentation in an offhand manner. Links I provide will seem unrelated at the first glance.